docker/Dockerfile.base

```diff
@@ -21,18 +32,33 @@ RUN git clone --depth=1 --branch=${haystack_version} https://github.com/deepset-
 WORKDIR /opt/haystack
 
 # Use a virtualenv we can copy over the next build stage
-RUN python -m venv --system-site-packages /opt/venv
+RUN ${python} -m venv --system-site-packages /opt/venv
 ENV PATH="/opt/venv/bin:$PATH"
 
 RUN pip install --upgrade pip && \
+    [ ${#PYTORCH} -gt 0 ] && VERSION='torch=='$PYTORCH'.*' ||  VERSION='torch'; pip install --no-cache-dir -U $VERSION --extra-index-url https://download.pytorch.org/whl/cu113 && \
```

Comment:
If we bump up the pytorch version, I think the URL for pytorch will also change to `cu116`. and the lines below will also have to change accordingly.

Reply:
How so @mayankjobanputra? I am serious because the wheel is hard coded to cu113 with `--extra-index-url https://download.pytorch.org/whl/cu113`

Reply:
I just ran the docker image building process and then ran it (see https://github.com/deepset-ai/haystack/pull/3701#issuecomment-1361197095), but maybe you are right. What am I overlooking?

Reply:
TL;DR - Your changes works and no worries.

Long Answer - 
I just got skeptical here because when I was trying to play around pytorch-1.13, it showed me following command on their website: `pip3 install torch torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/cu116`

and I thought pytorch wouldn't have provided packages for older CUDA. But I went to `cu113` fetch URL and confirmed that it does exist so no worries :)

Reply:
Yes, it turns out you were right. The cu113 for torchvision and torchaudio is still missing from the distro. I had to revert back to 1.12.1. Thanks @mayankjobanputra